app/src/main/java/org/mozilla/fenix/settings/wallpaper/Extensions.kt

```diff
@@ -0,0 +1,21 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.wallpapers
+
+/**
+ * The extension function to group wallpapers according to their name.
+ **/
+
+fun List<Wallpaper>.groupByCollection(): Map<Wallpaper.Collection, List<Wallpaper>> = groupBy {
+    it.collection
+}.filter {
+    it.key.name != "default"
```

Comment:
May be we could use `org.mozilla.fenix.wallpapers.Wallpaper.Companion.defaultName` to reduce the number of hard-coded elements? I am trying to come up with the way to replace `classic-firefox`, but fail.

Reply:
@mavduevskiy , we can pick this up in a follow up PR